site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -21,7 +21,7 @@
 
     <div id="student_download_view">
         <div>
-            <fieldset><legend>View and download submitted files?</legend>
+            <fieldset><legend>View and download submitted files? <span class="warning-text" id="warning"> WARNING: Students will see the grading score and TA feedback but will not be able to download the scanned and uploaded file.</span></legend>
```

Comment:
Let's change the class here to use `red-message` which is already defined in `site/public/css/server.css`, this would make the message look like how the other warning/error messages look like - it would also be needed to meet our accessibility standards for the amount of contrast required.

![image](https://user-images.githubusercontent.com/12129065/217115434-65d98350-9dca-4f98-952c-59cadd293774.png)